### PR TITLE
Merge Repos

### PR DESCRIPTION
**Pull request by Moux**
Merge the Rover Campaigns and Rover Experience SDK as the Rover SDK.

Resolves https://github.com/judoapp/rover-issues/issues/13

### CODE REVIEW

File: example-app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <resources>
-    <string name="app_name">Rover Example</string>
-</resources>
+    <string name="app_name">Rover  Example App</string>
```
Comment (orospakr) [inline example-app/src/main/res/values/strings.xml]: weird space here. artifact of a batch find replace?

File: example-app/src/main/res/values/constants.xml [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
-    <string name="associated_domain">blank</string>
+
+    <!-- This is the domain associated with your account in Rover Settings.  Rover can host
+    Experiences with App Links (Universal Links) that can be opened in a web browser or your
+    suitably enabled app. -->
+    <string name="rover_associated_domain">rover-labs-inc.rover.io</string>
+
+    <!-- The Rover Campaigns SDK has several built-in deep link handlers.  This field should be set
+    to match the URL Scheme set in your Rover Settings. -->
+    <string name="rover_uri_scheme">rv-rover-labs-inc</string>
```
Comment (orospakr) [inline example-app/src/main/res/values/constants.xml]: hey it looks like I accidentally (sigh, classic) committed these settings into git.

You can revert it by just copying this version of the file over top of this one (removing "campaigns" natch) https://github.com/RoverPlatform/rover-campaigns-android/blob/49b628654b495bd9e5dcd0d06284b2f55b2fe8c1/example-app/src/main/res/values/constants.xml

File: README.md [hunk 3]
```diff
@@ -27,8 +27,12 @@ block.
 ```groovy
 dependencies {
     // ...
-    implementation "io.rover:sdk:3.8.2"
+    implementation "io.rover.sdk:core:4.0.0"
```
Comment (orospakr) [inline README.md:30]: "3.0" at the top of this file

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -34,5 +34,5 @@ jobs:
     - name: Build with Gradle
       env:
           DEPLOY_MAVEN_PATH: /tmp/m2
```
Comment (orospakr) [inline .github/workflows/ci.yml:36]: I think you missed deploy.yml.  Check the Rover Campaigns deploy.yml

**Review by orospakr — CHANGES_REQUESTED** [review]

File: example-app/src/main/res/values/strings.xml (hunk 1 above)
Comment (Moux) [reply example-app/src/main/res/values/strings.xml]: Probably.  Will fix.

File: example-app/src/main/res/values/constants.xml (hunk 2 above)
Comment (Moux) [reply example-app/src/main/res/values/constants.xml]: ok.

File: README.md (hunk 3 above)
Comment (Moux) [reply README.md:30]: bah, will fix.

File: .github/workflows/ci.yml (hunk 4 above)
Comment (orospakr) [reply .github/workflows/ci.yml:36]: deploy.yml still missing

**Review by orospakr — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 4 above)
Comment (Moux) [reply .github/workflows/ci.yml:36]: The Rover Campaigns deploy.yml is identical to the one here, save the references ro rover-campaigns-android.

Comment (Moux) [reply .github/workflows/ci.yml:36]: TO be clear, there already was a deploy.yml in the master branch, and it didn't really have changes from the one in the old Rover Campaigns repo, save the references to rover-campaigns-android.   That is why it isn't in the PR.  There is no diff.

Comment (Moux) [reply .github/workflows/ci.yml:36]: Sorry, was wrong, the gradle line DID need changing.

**Review by orospakr — APPROVED** [review]

**Pull request merged**